gulpfile.js

```diff
@@ -263,7 +263,8 @@
 		 * Specifies the default task to run if no task is passed in.
 		 */
 		gulp.task("default", gulp.series("dev:run"));
-
+		/** Launches the application without building the files. For tsting prod builds. */
+		gulp.task("dev:no-build", gulp.series(taskMethods.startServer, taskMethods.launchApplication))
```

Comment:
Task should be called start:no-build or some such. The methods will use the environment specified in NODE_ENV so dev is a misnomer.